src/qt/walletmodel.cpp

```diff
@@ -139,7 +139,10 @@ void WalletModel::pollBalanceChanged()
             transactionTableModel->updateConfirmations();
 
         // check sigma
-        checkSigmaAmount(false);
+        // support only hd
+        if (pwalletMain->IsHDSeedAvailable()) {
```

Comment:
Will be better to checking in `checkSigmaAmount` instead so it will not need to check everywhere that invoked it.